decode.rb

```diff
@@ -0,0 +1,57 @@
+MORSE_CODE = {
+  '.-' => 'A',
+  '-...' => 'B',
+  '-.-.' => 'C',
+  '-..' => 'D',
+  '.' => 'E',
+  '..-.' => 'F',
+  '--.' => 'G',
+  '....' => 'H',
+  '..' => 'I',
+  '.---' => 'J',
+  '-.-' => 'K',
+  '.-..' => 'L',
+  '--' => 'M',
+  '-.' => 'N',
+  '---' => 'O',
+  ' .--.' => 'P',
+  '--.-' => 'Q',
+  '.-.' => 'R',
+  '...' => 'S',
+  '-' => 'T',
+  '..-' => 'U',
+  '...-' => 'V',
+  '.--' => 'W',
+  '-..-' => 'X',
+  '-.--' => 'Y',
+  ' --..' => 'Z'
+}.freeze
+
+def decode_char(str)
+  if MORSE_CODE.key?(str)
+    MORSE_CODE[str]
+  else
+    ''
+  end
+end
+
+def decode_word(str)
+  word = ''
+  str.split.each do |letter|
+    word += MORSE_CODE[letter]
+  end
```

Comment:
- [ ] We need to reuse the `decode_char` method we have here. Each character in a word needs to be decoded using the implemented method. Kindly do that.

  ```
  word += decode_char(letter)
  ```